docs/data/tree-view/overview/overview.md

```diff
@@ -75,7 +75,29 @@ Or increasing the width of the item state indicator to be full-width:
 
 ## Customization
 
-### Custom icons, border and animation
+### Custom icons
+
+You can customize the collapse, expand and end icons of the TreeView.
+
+You can use icon liraries:
+
+{{"demo": "CustomMaterialIcons.js", "defaultCodeOpen": true}}
+
+Or you can create your own components:
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Adding borders
+
```

Comment:
```suggestion
Target the `treeItemClasses.group` class to add connection borders between the Tree View items.
```

Guess we can have a simple description here so readers can understand _how_ they'd add a border even before expanding the code block.

Reply:
Rephrased a bit, because my approach there is a bit different, but adding an explanation is a good idea :)